### ISSUE TITLE
After pressing enter on a search result in Google NVDA announces the text of the link again and also reads its URL

### ISSUE THREAD

**Issue by lukaszgo1** [nvaccess/nvda#8895]
### Steps to reproduce:
1. Open Firefox.
2. Type something into the search box for example test.
3. On the newly opened Google page press 3 to move the first search result and press enter.
### Actual behavior:
NVDA repeats the text of the link and also reads the URL of it.
### Expected behavior:
NVDA should be silent or optionally say visited like it used to.
#### NVDA Installed/portable/running from source:
Installed
#### NVDA version:
alpha-16203,b4e9e832

#### Windows version:
Windows 7 x64
#### Name and version of other software in use when reproducing the issue:
Nightly 65
#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA?

Yes, NVDA 2018.3.2 and the problem wasn't present.

**Comment by jscholes** [comment]
Can't reproduce this.  Possibly an issue with your browser configuration, Internet connection, DNS provider or similar.

**Comment by jscholes** [comment]
Thanks for putting in the work to track it down.  I can reproduce it after updating to a newer alpha snapshot, which I probably should've done earlier :)

**Comment by lukaszgo1** [comment]
@michaelDCurran Can you please look into this?

**Comment by leonardder** [comment]
@michaeldcurran: I've also seen cases where browsing with the browse mode caret made a button being announced twice.

**Comment by jcsteh** [comment]
While we aren't seeing this since #8869 got reverted, I have an update on the underlying issue here.

It turns out it wasn't Mozilla bug 659886 (now fixed) after all, even though it manifests the same way. I filed https://bugzilla.mozilla.org/show_bug.cgi?id=1570540 for this particular Google Search case, which is related to the href attribute changing.

**Comment by jcsteh** [comment]
Fixed in Firefox 77.

**Comment by lukaszgo1** [comment]
Given the fact that it is fixed can we reintroduce #8869?

**Comment by jcsteh** [comment]
As I understand it, this wasn't the only issue that caused #8869 to be reverted. However, several other issues have also been fixed since then. Probably a question for @MichaelDCurran.
